### PR TITLE
Upload vade builds for different targets on release

### PR DESCRIPTION
**Pull request by vineetpant**
## Description

This Pr builds vade-evan for bundle-sdk feature on following targets and upload them to release artifacts:
            # linux builds
            x86_64-unknown-linux-gnu,
            aarch64-unknown-linux-gnu,
            arm-unknown-linux-gnueabihf,
            # macos builds
            x86_64-apple-darwin,
            aarch64-apple-darwin,
            # windows builds
            x86_64-pc-windows-msvc,
            x86_64-pc-windows-gnu,
            # android builds
            aarch64-linux-android,
            armv7-linux-androideabi,
            x86_64-linux-android,
            i686-linux-android,
            # ios builds
            aarch64-apple-ios,
            x86_64-apple-ios,
            aarch64-apple-ios-sim,
            wasm32-unknown-unknown,
## Details 
-  build for various targets 
- upload static lib for various targets to release artifact

### CODE REVIEW

**Review by Tschuck — APPROVED** [review]
LGTM. But please wait for second approval.

**Review by S3bb1 — APPROVED** [review]

**Review by wulfraem — APPROVED** [review]

**Pull request merged**
